tools/sof-kernel-log-check.sh

```diff
@@ -21,8 +21,9 @@ ignore_str="$ignore_str"'|mei_me 0000:00:16\.0: wait hw ready failed'
 ignore_str="$ignore_str"'|mei_me 0000:00:16\.0: hw_start failed ret = -62'
 
 # On CML_RVP_SDW, suspend-resume test case failed due to "mei_me 0000:00:16.4: hw_reset failed ret = -62".
+# The PCI ID in latest sof-dev differs from the PCI ID in linux-5.8.y
 # https://github.com/thesofproject/sof-test/issues/389
-ignore_str="$ignore_str"'|mei_me 0000:00:16\.4: hw_reset failed ret = -62'
+ignore_str="$ignore_str"'|mei_me 0000:00:16\..: hw_reset failed ret = -62'
```

Comment:
maybe it is better to ignore the exact pci id here and match it with a pattern?

Reply:
I think a `.` here is good enough.